vnext/Microsoft.ReactNative/Fabric/Composition/ActivityIndicatorComponentView.h

```diff
@@ -58,6 +58,8 @@ struct ActivityIndicatorComponentView
   winrt::Microsoft::ReactNative::Composition::ISpriteVisual m_visual{nullptr};
   winrt::Microsoft::ReactNative::Composition::IActivityVisual m_ActivityIndicatorVisual{nullptr};
   facebook::react::SharedViewProps m_props;
+  float m_radiusSmall = 8.0f;
+  float m_radiusLarge = 16.0f;
```

Comment:
These values shouldn't change, right? So they should be declared as constants?

```suggestion
  constexpr float m_radiusSmall = 8.0f;
  constexpr float m_radiusLarge = 16.0f;
```

At which point giving them a `m_` prefix wouldn't really be the standard. Either classic `k_radiusSmall` for constant or `RadiusSmall` or `RADIUS_SMALL` or whatever the style guide is here for constants.